lib/services/stapi-service.js

```diff
@@ -0,0 +1,20 @@
+import request from 'superagent';
+import TREK from '../models/stapi-model';
+//import stapi_model from '../models/stapi-model';
+
+export default class trekService {
+
+    static async getTrekSeasons(character) {
```

Comment:
This is a great use of a service! We had been focusing a lot on adding things on the service level that get returned to the user, but adding things in the service that get included in the shape of the data that gets used in the model is also a super common use case.